core/iwasm/libraries/lib-wasi-threads/test/build.sh

```diff
@@ -9,10 +9,13 @@ set -eo pipefail
 CC=${CC:=/opt/wasi-sdk/bin/clang}
 WAMR_DIR=../../../../..
 
+# Stress tests names
+stress_tests=("spawn_stress_test.wasm")
```

Comment:
is it only used to decide whether the `thread_start_file` is needed or not?  If so, maybe worth renaming this array and include `linear_memory_size_update` in it.